composer.json

```diff
@@ -18,10 +18,10 @@
         }
     },
     "require": {
-        "php": ">=5.5.0",
+        "php": ">=8.1",
```

Comment:
IMO this should be `^8.1` instead since you can impossibly make a promise that your package will work with all future PHP versions which will ever exist in the history of mankind. Also mentioned in [The Death Star Version Constraint](https://thephp.cc/articles/the-death-star-version-constraint)